lib/util/error.js

```diff
@@ -968,6 +968,11 @@ shaka.util.Error.Code = {
    */
   'MISSING_STORAGE_CELL': 9013,
 
+  /**
+   * The allowed storage limit has been reached.
```

Comment:
I think it would be better to word this in terms of the callback.  This suggests we have our own limit.

Reply:
Done!